### PR TITLE
[v3.8.5] Fix Model.setInstancedAttribute doesn't support attribute of INT format.

### PR DESCRIPTION
**Pull request by GengineJS**

### CODE REVIEW

File: native/cocos/bindings/manual/jsb_scene_manual.cpp [hunk 1]
```diff
@@ -709,9 +709,9 @@ static bool js_Model_setInstancedAttribute(se::State &s) // NOLINT(readability-i
                 switch (type) {
                     case se::Object::TypedArrayType::FLOAT32: {
                         uint8_t *data = nullptr;
-                        size_t byteLength = 0;
-                        if (val.toObject()->getTypedArrayData(&data, &byteLength) && data != nullptr && byteLength > 0) {
-                            cobj->setInstancedAttribute(name, reinterpret_cast<const float *>(data), static_cast<uint32_t>(byteLength));
+                        size_t byteCount = 0;
```
Comment (star-e) [inline native/cocos/bindings/manual/jsb_scene_manual.cpp]: I think `byteCount` means `elementCount`？
Becasue `byteCount` is no different from `byteLength` by name.

Comment (dumganhar) [reply native/cocos/bindings/manual/jsb_scene_manual.cpp]: getTypedArrayData returns byte length of the internal buffer. 
Why change the variable name from `byteLength` to `byteCount`?

**Comment by qiuguohua** [comment]
https://github.com/cocos/cocos-engine/issues/17859

**Comment by dumganhar** [comment]
@cocos-robot run test cases

**Review by dumganhar — APPROVED** [review]

**Review by star-e — APPROVED** [review]

**Review by qiuguohua — APPROVED** [review]

**Review by minggo — APPROVED** [review]

**Pull request merged**